### ISSUE TITLE
setup not working correctly

### ISSUE THREAD

**Issue by kdere** [chianti-atomic/ChiantiPy#95]
for example
```
k4=ch.ion('k_4', 1.e+5, 1.e+9, setup=0)
k4.setup(alternate_dir='.')
```
gives
elvlc file does not exist:  /data2/svn/chianti/dbase/trunk/k/k_4/k_4.elvlc

it should look in the local directory (pwd/cwd) to find the files

**Comment by kdere** [comment]
the problem was that k_4 was not in the masterlist
I will have fix this somehow

**Comment by kdere** [comment]
fixed by #96
